src/dataflow/analysis/flowconfig.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+/**
+ * A FlowConfig is a set of assertions used to configure a FlowChecker,
+ * along with any assertion-related metadata. The language for specifying
+ * the assertions is parsed in the constructor, or, if we choose to have
+ * a FlowAssertion class, each assertion is parsed in the FlowAssertion
+ * constructor.
+ */
+export class FlowConfig {
+  public assertions: string[]; // parsed assertions usable by FlowChecker
```

Comment:
Is string the right format for a parsed assertion?

Reply:
No. It's a stand-in for something more complete to come. I just wasn't sure I wanted to create a FlowAssertion class, yesterday, but on thinking it through overnight I've decided that I do, so the next PR will have one.